### ISSUE TITLE
run pa11y tests on dev url

### ISSUE THREAD

**Issue by harrisonpim** [harrisonpim/theblackhart.co.uk#69]
- [x] sitemap needs to be dynamically generated
- [x] pa11y needs to run on the locally built version of the sitemap in ci

sitemap script should generate urls based on an environment variable: localhost in CI and theblackhart.co.uk in prod.
pa11y action should build, export, and serve the current version of the code, and Pa11y-ci should then be run against the local sitemap. 

this seems like it might be the right place to look?
https://glebbahmutov.com/blog/develop-preview-test/
